core/src/components/toast/toast.ios.scss

```diff
@@ -30,6 +30,10 @@
     background: $toast-ios-translucent-background-color;
     backdrop-filter: $toast-ios-translucent-filter;
   }
+
+  :host(.ion-color.toast-translucent) .toast-wrapper {
+    background: rgba(var(--ion-color-base-rgb, 255, 255, 255), $toast-ios-translucent-background-color-alpha);
```

Comment:
What was the reasoning to not use the `current-color` [function](https://github.com/ionic-team/ionic-framework/blob/d3232dcc007738fae851660bd818c8056b48b7d2/core/src/themes/ionic.functions.color.scss#L7)?

```scss
:host(.ion-color.toast-translucent) .toast-wrapper {
  background: #{current-color(base, $toast-ios-translucent-background-color-alpha)};
}
```

Reply:
only because i didn't realize that `current-color` can accept an alpha 🙃